cadc-file-validate/README.md

```diff
@@ -0,0 +1,74 @@
+# cadc-file-validate 0.1
+
```

Comment:
In the model this component is simply denoted as a _process_ without saying exactly how it is implemented... some of them may run and terminate like a command line tool and others may run forever and manage their own idle time... better to call it a something vague like _process_ or _tool_ but not command-line.